components/homeComponents/HomeChallengeCard.tsx

```diff
@@ -1,32 +1,21 @@
-/**
- * Challenge Cards Component
- *
- * Cards for challenge section in home page
- * To add a linebreak for the description, simply add \n into the string value where needed in firebase
- */
-
-function HomeChallengeCard(props) {
-  var description;
-  if (props.description !== undefined && props.description !== null) {
-    description = props.description.replaceAll('\\n', '\n');
-  }
+export default function HomeChallengesCard(props: { challenge: Challenge; blockType: number }) {
+  const borderConfiguration = ['rounded-tr-[100px]', 'rounded-br-[100px]', 'rounded-tl-[100px]'];
   return (
-    <div className="bg-primary/50 w-full rounded-lg p-6 text-primaryDark">
-      <div className="text-left font-bold md:text-2xl test-base">{props.title}</div>
-      <div className="text-left md:text-xl test-base">Presented by {props.organization}</div>
-      <p className="whitespace-pre-line my-3 md:text-base text-xs">{description}</p>
-      {props.prizes !== null && props.prizes !== undefined && (
-        <div className="md:text-base text-sm">
-          <div className="underline font-medium md:text-base text-sm">Prizes</div>
-          <ul className="list-decimal list-inside md:text-base text-sm">
-            {props.prizes.map((prize, idx) => (
-              <li key={idx}>{prize}</li>
-            ))}
-          </ul>
+    <div className="w-full h-[200px] mb-[160px]">
+      <div className="mx-auto">
+        {/* Block */}
+        <div className={`bg-[#C1C8FF] ${borderConfiguration[props.blockType]} w-4/5 h-[200px]`}>
+          &nbsp;
         </div>
-      )}
+        {/* Challenge Name */}
+        <h1 className="text-2xl font-medium">{props.challenge.title.toUpperCase()}</h1>
+        {/* Company Name */}
+        <h1 className="text-xl text-[#05149C]">{props.challenge.organization}</h1>
+        {/* Description */}
+        <div className="h-[200px]">
+          <p className="text-md line-clamp-5 text-balance">{props.challenge.description}</p>
+        </div>
```

Comment:
The spacing between the lines of text could be increased. And the font weight and family for the title + description could be more bold. Otherwise, looks good👍